### PR TITLE
Feature/filter restaurants

### PR DESCRIPTION
**Pull request by vinayaknatarajan**
Added the following Models:
- YelpRestaurantDetails: To save Yelp restaurant details for implementing filter feature.
- Zipcodes: to store mappings for neighborhoods

Added module to query Yelp business categories API to get cuisine type for the restaurant.

Updated DateTime format on the inspection history view for better readability:

Added script to query yelp and save the restaurant attributes on which filtering can be performed.

Note: @deusalexmachina  @chenwei713  Please review and suggest if any changes are required in the above models.

### CODE REVIEW

**Comment by deusalexmachina** [comment]
@vinayaknatarajan, seems like Yelp-related tests are failing due to reaching the API limit. We should implement the API swapper to avoid these problems.

File: neighborhoodmap.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+def get_cuisine(categories):
+
+    for c in categories:
+        category = get_restaurant_category_yelp(c["alias"])
+        if category["category"]["parent_aliases"][0] == "restaurants":
+            return c["alias"]
+        else:
+            return category["category"]["parent_aliases"][0]
+        # break
```
Comment (deusalexmachina) [inline neighborhoodmap.py]: remove comment

**Comment by deusalexmachina** [comment]
Also, how is coverage? We could try to write some tests before we merge pull requests as this will impact our build status.

**Comment by deusalexmachina** [comment]
> @vinayaknatarajan, seems like Yelp-related tests are failing due to reaching the API limit. We should implement the API swapper to avoid these problems.

actually, I believe we could just use a separate API Token for our tests. Thoughts?

File: restaurant/views.py [hunk 2]
```diff
@@ -47,6 +47,11 @@ def get_inspection_info(request, restaurant_id):
             restaurant.restaurant_name, restaurant.business_address, restaurant.postcode
         )
 
+        for i in range(len(inspection_data_list)):
+            inspection_data_list[i]["inspected_on"] = inspection_data_list[i][
```
Comment (chenwei713) [inline restaurant/views.py]: it could be like:
for inspection_data in inspection_data_list:

File: restaurant/views.py [hunk 3]
```diff
@@ -47,6 +47,11 @@ def get_inspection_info(request, restaurant_id):
...
+                "inspected_on"
```
Comment (chenwei713) [inline restaurant/views.py]: And I think it's better to put this code to utils

**Review by deusalexmachina — APPROVED** [review]

**Review by deusalexmachina — APPROVED** [review]

**Pull request merged**